target/scripts/start-mailserver.sh

```diff
@@ -41,12 +45,17 @@ VARS[FAIL2BAN_BLOCKTYPE]="${FAIL2BAN_BLOCKTYPE:=drop}"
 VARS[FETCHMAIL_PARALLEL]="${FETCHMAIL_PARALLEL:=0}"
 VARS[FETCHMAIL_POLL]="${FETCHMAIL_POLL:=300}"
 VARS[LDAP_START_TLS]="${LDAP_START_TLS:=no}"
-VARS[LOGROTATE_INTERVAL]="${LOGROTATE_INTERVAL:=${REPORT_INTERVAL:-daily}}"
+VARS[LOGROTATE_INTERVAL]="${LOGROTATE_INTERVAL:=daily}"
```

Comment:
```suggestion
VARS[LOGROTATE_INTERVAL]="${LOGROTATE_INTERVAL:=weekly}"
```

Reply:
Now I see committing the suggestion in https://github.com/docker-mailserver/docker-mailserver/pull/2428#discussion_r812365051 was an error on my side. We can go ahead and change the default of `LOGROTATE_INTERVAL` to weekly, but then we'd need to do it uniformly. The old default value was `daily`. If you agree @casperklein, I will change it to weekly (then I'd also need to change the docs and some tests). Otherwise, if we want to stay at the current default, it'd need to revert https://github.com/docker-mailserver/docker-mailserver/pull/2428#discussion_r812365051 for consistency.

Reply:
The logs are rotated 4 times. I think `weekly` is a better default, otherwise there isn't a complete week of logs. This is the minimum, one should have in case of errors for debugging.

Reply:
Then I will change it consistently to `weekly` with the next commit.

Reply:
resolved in [ad7e1f7](https://github.com/docker-mailserver/docker-mailserver/pull/2428/commits/ad7e1f72bafdad7f5dbbb94d05f23cfb06ffc93d)